dist/images/ovnkube.sh

```diff
@@ -1012,7 +1012,7 @@ local-sb-ovsdb() {
 
 # v1.0.0 - Runs northd on master. Does not run nb_ovsdb, and sb_ovsdb
 run-ovn-northd() {
-  trap 'ovs-appctl -t ovn-northd exit >/dev/null 2>&1; exit 0' TERM
+  trap 'ovn-appctl -t ovn-northd exit >/dev/null 2>&1; exit 0' TERM
```

Comment:
not an expert here regarding shutting down northd. I see [1] ovs-appctl used in the man page.
How do you know northd isnt gracefully stopped?

[1] https://man7.org/linux/man-pages/man8/ovn-northd.8.html

Reply:
cc @dceara  WDYT?

Reply:
I see.
ovn-appctl and ovs-appctl do the same thing when the target is the same. I didn't know that.

I was using the "ovnkube.sh" script in a different environment than ovn-kubernetes, and didn't mount /var/run/ovn and /var/run/openvswitch to the same directory, so ovs-appctl couldn't find the target of "ovn-northd" in /var/run/openvswitch because it's in the only /var/run/ovn directory.

I tried graceful shutdown in the environment created by ovn-kubernetes helm chart, it worked correctly. So there was no problem. I was wrong.

Thanks for your work for clarity.